data/stackbit.json

```diff
@@ -2150,44 +2150,44 @@
   "stackbit-themes-stackbit-starter-jekyll-master": {
     "createUrl": "https://app.stackbit.com/create?theme=https://github.com/stackbit-themes/stackbit-starter-jekyll&ssg=jekyll"
   },
-  "stackbithq-stackbit-theme-ampersand-master": {
-    "createUrl": "https://app.stackbit.com/create?theme=https://github.com/stackbithq/stackbit-theme-ampersand"
+  "stackbit-stackbit-theme-ampersand-master": {
```

Comment:
@JugglerX Please specifically review the rename of these keys. Will changing these keys break anything related to analytics?

Reply:
@ramonsnir I fixed these directly in another commit, so closing this PR. Thanks for the heads up.